test/src/test/java/org/corfudb/runtime/concurrent/CorfuQueueTxTest.java

```diff
@@ -32,6 +32,8 @@
 import org.corfudb.runtime.exceptions.TransactionAbortedException;
 import org.corfudb.runtime.object.transactions.AbstractTransactionsTest;
 import org.corfudb.runtime.object.transactions.TransactionType;
+import org.corfudb.runtime.proto.RpcCommon;
```

Comment:
nit: These unused imports could be removed.

Reply:
Done.